JS-Algorithms/challenges.js

```diff
@@ -118,9 +150,24 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
-};
+  let n1 = 0, n2 = 1, nextTerm;
+  let result = [0, 1]
+
+
+  nextTerm = n1 + n2;
+
+  while(true) {
```

Comment:
when use `while(true)`, be sure to end this loop in some moment, and if its possible, pass that end condition to the `while` statement